aca_preview/preview.py

```diff
@@ -336,3 +340,66 @@ def check_bright_guide_for_ers(self, n_bright_req=3, bright_lim=9.0):
         n_bright = np.count_nonzero(self.guides['mag'] < bright_lim)
         if not is_OR and n_bright < n_bright_req:
             self.add_message('critical', f'ER bright stars: only {n_bright} stars brighter than {bright_lim}')
+
+    def check_enough_guide_for_ers(self, n_required=8):
+        """
+        Warn on ERs with fewer than n_required (8) guide stars.
+        """
+        obsid = float(self.obsid)
+        is_OR = abs(obsid) < 38000
```

Comment:
Make an `is_OR` property to stop this repetition.  And while you are at it might as well put in an `is_ER` property that returns `not is_OR`.  This just makes downstream checks more obvious.

Reply:
Where is the most sensible place to set the value of the new properties?

Reply:
I guess as long as obsid is set reasonably, I don't need a place.

Reply:
Right, as long as obsid is set then the property is just read-only whenever needed.  No need to even store it, just compute on the fly each time.